src/main/kotlin/io/kuzzle/sdk/coreClasses/exceptions/ApiErrorException.kt

```diff
@@ -30,8 +30,8 @@ class ApiErrorException : KuzzleException {
     constructor(response: Response) :
         super(response.error?.message, response.status) {
             if (response.error != null) {
-                this.stack = response.error?.stack
-                this.id = response.error?.id
+                this.stack = response.error!!.stack
+                this.id = response.error!!.id
```

Comment:
you really need the "!!" in this if since you have "response.error != null" above ?

Reply:
Yes the compiler woudn't let me do that even though I checked for null safety before

Reply:
Ok, it's sad to have a null safety language like Kotlin and be forced to do this :'(.